library/src/main/scala/scala/scalajs/runtime/Bits.scala

```diff
@@ -16,9 +16,18 @@ import js.typedarray
 /** Low-level stuff. */
 object Bits {
 
-  val areTypedArraysSupported = js.DynamicImplicits.truthValue(
-      global.ArrayBuffer && global.Int32Array &&
-      global.Float32Array && global.Float64Array)
+  private[this] val _areTypedArraysSupported = {
```

Comment:
I would add the following comment to `areTypedArraysSupported`:

```
We have a forwarder to the internal val _areTypedArraysSupported to be able to inline it. This achieves the following:

- Completely DCE `_areTypedArraysSupported` if we emit ES6 and replace `areTypedArraysSupported` by `true` in the calling code
- If we emit ES5 replace `areTypedArraysSupported` by `_areTypedArraysSupported` so we do not calculate it multiple times.
```

I think that could help in understanding why there is a forwarder.